### PR TITLE
Remove flux binary from build image

### LINKED ISSUE #4577: flux integration test is failing from time to time
The latest test run failed with this error. We haven't encountered this one yet.

```
2022-01-06T11:05:26.3318719Z [13] error: unable to recognize "/tmp/flux-bootstrap-2411656037/flux-system": no matches for kind "Kustomization" in version "kustomize.toolkit.fluxcd.io/v1beta2"
2022-01-06T11:05:26.3384931Z [13] ‚úó exit status 1
2022-01-06T11:05:26.3427237Z [13] 2022-01-06 11:05:26 [‚Ñπ]  Flux v2 failed to install successfully. check configuration and re-run `eksctl enable flux`
```

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Closes #4577

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -116,28 +116,6 @@ build-integration-test: $(all_generated_code) ## Ensure integration tests compil
 integration-test: build build-integration-test ## Run the integration tests (with cluster creation and cleanup)
 	JUNIT_REPORT_DIR=$(git_toplevel)/test-results ./eksctl-integration-test $(INTEGRATION_TEST_ARGS)
 
-# NOTE: when running this, ensure all env vars listed below are exported in your local env
-# SSH_KEY_PATH should be set to wherever you have locally put the eksctl-bot rsa key
-.PHONY: integration-test-container
-integration-test-container: ## Run integration tests in a local container
-	sudo cp -f ${SSH_KEY_PATH} $(HOME)/project/.ssh/gitops_id_rsa
```
Comment (aclevername) [inline Makefile:123]: delete the docs too

Comment (Skarlso) [reply Makefile:123]: Ah cool

Comment (Skarlso) [reply Makefile:123]: Is that all? The contribution guide?

Comment (aclevername) [reply Makefile:123]: think so

**Review by aclevername — APPROVED** [review]

**Pull request merged**